mozilla/blurts-server
# Stop recreating Animation component every render

# References:

Jira: MNTOR-3348
Figma: N/A



# Description

This caused a flicker where the video gets reloaded, and since re-renders happen whenever the session is checked, this happens fairly frequently: about once every minute, or when switching tabs.

# How to test

Go to the settings page, open the Network Console and filter it on `session`. If you do this on `main`, you'll see a flicker whenever you see a new request going there (which you can trigger by switching tabs and back). That should no longer happen on this branch.

# Checklist (Definition of Done)

- [ ] I've added a unit test to test for potential regressions of this bug. - N/A, I don't think we can recreate the browser video loading behaviour in tests.
